docs/running-headscale-container.md

```diff
@@ -6,72 +6,98 @@
     written by community members. It is _not_ verified by `headscale` developers.
 
     **It might be outdated and it might miss necessary steps**.
-
+    
+    Last update : 01/05/2023 - Headscale 0.22.1
+ 
 ## Goal
 
-This documentation has the goal of showing a user how-to set up and run `headscale` in a container.
-[Docker](https://www.docker.com) is used as the reference container implementation, but there is no reason that it should
-not work with alternatives like [Podman](https://podman.io). The Docker image can be found on Docker Hub [here](https://hub.docker.com/r/headscale/headscale).
+This documentation has the goal of showing a user how-to set up and run `headscale` in a [Docker](https://www.docker.com) container.
+
+The Docker image can be found on Docker Hub [here](https://hub.docker.com/r/headscale/headscale).
 
 ## Configure and run `headscale`
 
-1. Prepare a directory on the host Docker node in your directory of choice, used to hold `headscale` configuration and the [SQLite](https://www.sqlite.org/) database:
+# 1. Prepare a directory on the host Docker node in your directory of choice, used to hold `headscale` files. 
 
 ```shell
-mkdir -p ./headscale/config
-cd ./headscale
-```
-
-2. Create an empty SQlite datebase in the headscale directory:
+mkdir -p ./headscale/config && cd ./headscale/config
+``` 
+    
+!!! info "Database"
+    
+    This tutorial uses SQLite for simplicity reasons, in a large environnement, it's recommended to use Postgres,  
+    please refer the configuration file
+    
+# 2. Create an empty [SQLite](https://www.sqlite.org/) datebase in the headscale directory:
 
 ```shell
 touch ./config/db.sqlite
 ```
 
-3. **(Strongly Recommended)** Download a copy of the [example configuration][config-example.yaml](https://github.com/juanfont/headscale/blob/main/config-example.yaml) from the headscale repository.
-
-Using wget:
+# 3. Create an ACL file in the headscale directory:
```

Comment:
Just point out that this is optional as the example-config does not point to a acl path.  Further down you add the according config. Good addition. Maybe add it as an optional point.

Reply:
Don't everybody want to use ACLs ? And I'm pretty sure that the docker image won't boot without acls, but I need to check

Reply:
It will boot without ACLs. I don't mean to remove it just, as an extra point that you can optionally add a acl. It is not needed for a basic setup.

Reply:
After a time thinking to it, I came up with that : Initial work is a good understanding way. I'm sure that pretty much no one will use Headscale without ACLs, and even if it's the case, if them want to use them later (they would probably want), it'll be more simple for them to just edit a fill than finding the solution in the

Reply:
What do you think ?

Reply:
This wording makes it sound like it is necessary, that is why I think it would be better it's own point and say it is optional.